1.0/docs/devguide/events.md

```diff
@@ -310,46 +291,62 @@ normalized event has the following properties:
     shady DOM).
 
 *   `localTarget`: Retargeted event target (equivalent to `event.target` under
-    shadow DOM)
+    shadow DOM). This node is always in the same scope as the node where the
+    listener was added.
 
 *   `path`: Array of nodes through which event will pass
     (equivalent to `event.path` under shadow DOM).
 
 
 Example:
+
+    <!-- event-retargeting.html -->
+     ...
     <dom-module id="event-retargeting">
       <template>
         <button id="myButton">Click Me</button>
       </template>
-    </dom-module>
-
-    <script>
-
-      Polymer({
-
-          is: 'event-retargeting',
-
-          listeners: {
-            'myButton.click': 'handleClick',
-          },
 
-          handleClick(e) {
-            console.info(e.target.id + ' was clicked.');
-          }
+      <script>
+        Polymer({
 
-        });
+            is: 'event-retargeting',
 
-    </script>
+            listeners: {
+              'myButton.click': 'handleClick',
+            },
 
+            handleClick(e) {
+              console.info(e.target.id + ' was clicked.');
+            }
+          });
+      </script>
     </dom-module>
 
+
+    <!-- index.html  -->
+      ...
     <event-retargeting></event-retargeting>
 
     <script>
-      document.querySelector('event-retargeting').addEventListener('click', function(){
-        var normalizedEventObject = Polymer.dom(event);
-        console.info('rootTarget is:', normalizedEventObject.rootTarget); // logs #myButton
-        console.info('localTarget is:', normalizedEventObject.localTarget); // logs an instance of event-targeting that was originally queried
-        console.info('path is:', normalizedEventObject.path); // logs [#myButton, document-fragment, event-retargeting, body, html, document, Window]
+      document.querySelector('event-retargeting').addEventListener('click',
+          function(){
+        var normalizedEvent = Polymer.dom(event);
```

Comment:
This is intentional--when wrapping a long function invocation, the next argument's supposed to be either indented 4 spaces or aligned with the parens. I don't love the way this looks, but I also don't love the alternatives...